Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "async-broadcast"
-version = "0.4.0"
+version = "0.4.1"
```

Comment:
This change should definitely go in a separate commit.

Reply:
@Antiarchitect Thanks for redoing. Just FYI, by separate commit, i mean only commit, no need for a separate PR.